toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -900,6 +902,32 @@ func customizeVerityImageHelper(buildDir string, config *imagecustomizerapi.Conf
 		return nil, fmt.Errorf("failed to get disk's (%s) sector size:\n%w", loopback.DevicePath(), err)
 	}
 
+	for _, metadata := range baseImageVerity {
```

Comment:
This root hash "replace only" logic is impressive.

Reply:
Thanks!